### PR TITLE
[ISSUE_146] - add upgrade/merge access methods

### PR DESCRIPTION
**Pull request by baranowb**
https://github.com/jboss-set/aphrodite/issues/146

### CODE REVIEW

File: domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package org.jboss.set.aphrodite.domain;
```
Comment (soul2zimate) [inline domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java]: header and copyright

File: domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public enum MergeableState {
```
Comment (soul2zimate) [inline domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java]: Are these definitions come from String mergeable_state of GHPullRequest ?

File: github/src/main/java/org/jboss/set/aphrodite/repository/services/github/GitHubWrapper.java [hunk 3]
```diff
@@ -64,22 +66,26 @@ Repository toAphroditeRepository(URL url, Collection<GHBranch> branches) {
...
-            String body = pullRequest.getBody() == null ? "" : pullRequest.getBody().replaceFirst("\\u2026", "");
-
+            final String id = Integer.toString(pullRequest.getNumber());
+            final URL url = pullRequest.getHtmlUrl();
+            final Codebase codebase = new Codebase(pullRequest.getBase().getRef());
+            final PullRequestState state = getPullRequestState(pullRequest.getState());
+            final String title = pullRequest.getTitle() == null ? "" : pullRequest.getTitle().replaceFirst("\\u2026", "");
+            final String body = pullRequest.getBody() == null ? "" : pullRequest.getBody().replaceFirst("\\u2026", "");
+            final boolean mergeable = pullRequest.getMergeable();
+            final boolean merged = pullRequest.isMerged();
```
Comment (soul2zimate) [inline github/src/main/java/org/jboss/set/aphrodite/repository/services/github/GitHubWrapper.java]: In previous EGit, it only counts "button click merge" and misses "push + close merge". I guess it's same in github-api.

File: domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java (hunk 2 above)
Comment (baranowb) [inline domain/src/main/java/org/jboss/set/aphrodite/domain/MergeableState.java]: Yes.

**Pull request merged**